contributors/design-proposals/storage/raw-block-pv.md

```diff
@@ -528,6 +528,40 @@ spec:
         name: my-nginx-data 
 	readOnly: false
 ```
+## UC9:
+
+DESCRIPTION:
+* A user wishes to read data from a read-only raw block device, such as a database for analytics processing. 
+
+ADMIN:
+* Admin creates a 1 block devices and sets container runtime to read-only mode using the --device option.
```

Comment:
Admin creates a raw block device.

>> and sets container runtime to read-only mode using the --device option.
This operations are unnecessary. Admin doesn't create container, Kubernetes/kubelet will create and set --device option

Reply:
yes...fixing